Modules/+Sources/@VelocityLaser/VelocityLaser.m

```diff
@@ -20,9 +20,9 @@
 
     properties
         prefs = {'PBline','pb_ip','velocity_ip','wavemeter_ip','wavemeter_channel',...
-                 'cal_local','TuningTimeout','TuneSetpointAttempts','TuneSetpointNPoints'};
+                 'wheel_ip', 'wheel_pin', 'wheel_pos', 'cal_local','TuningTimeout','TuneSetpointAttempts','TuneSetpointNPoints'};
         show_prefs = {'PB_status','tuning','diode_on','wavemeter_active','PBline','pb_ip',...
-            'velocity_ip','wavemeter_channel','wavemeter_ip','TuningTimeout','TuneSetpointAttempts','TuneSetpointNPoints','debug'};
+            'velocity_ip','wavemeter_ip','wavemeter_channel','wheel_ip', 'wheel_pin', 'wheel_pos', 'TuningTimeout','TuneSetpointAttempts','TuneSetpointNPoints','debug'};
```

Comment:
"wavemeter_host"

Reply:
I know you probably didn't add that, but would be nice to change it throughout the file as well.